pandas/core/indexes/base.py

```diff
@@ -2022,48 +2022,18 @@ def hasnans(self):
 
     def isna(self):
         """
-        Detect missing values.
-
-        Return a boolean same-sized object indicating if the values are NA.
-        NA values, such as None or :attr:`numpy.NaN`, get mapped to True
-        values.
-        Everything else get mapped to False values. Characters such as empty
-        strings `''` or :attr:`numpy.inf` are not considered NA values
-        (unless you set :attr:`pandas.options.mode.use_inf_as_na` `= True`).
+        Detect missing values
 
         .. versionadded:: 0.20.0
 
         Returns
         -------
-        numpy.ndarray
-            A boolean array of whether my values are NA
+        a boolean array of whether my values are NA
```

Comment:
The right format should probably be
```
numpy.ndarray
    Boolean array of whether my values are NA.
```

Though I'd advise against "my values" and refer to the data structure.

> The documentation of the return is also similar to the parameters. But in this case, no name will be provided, unless the method returns or yields more than one value (a tuple of values).

> The parameters are defined by their name, followed by a space, a colon, another space, and the type (or types). Note that the space between the name and the colon is important. Types are not defined for *args and **kwargs, but must be defined for all other parameters. After the parameter definition, it is required to have a line with the parameter description, which is indented, and can have multiple lines. The description must start with a capital letter, and finish with a dot.

Reply:
Please ignore, was from incorrect commit